packages/ui-kit/src/blocks/deprecations.ts

```diff
@@ -35,6 +36,8 @@ export type IMultiStaticSelectElement = InterfaceOf<MultiStaticSelectElement>;
 /** @deprecated */
 export type IDatePickerElement = InterfaceOf<DatePickerElement>;
 /** @deprecated */
+export type IDateTimePickerElement = InterfaceOf<DateTimePickerElement>;
```

Comment:
I think that this Interface should not be on the deprecations

Reply:
The existing `datepicker`'s interface is also in deprecation, is there any reason for it?
Shall I remove new `datetimepicker`'s interface from deprecation?